iai_apartment/urdf/apartment.urdf

```diff
@@ -14,330 +14,389 @@
+        <geometry>
+          <box size="0.7 0.4 0.32" />
+        </geometry>
+      </collision>
+      <collision>
+        <origin rpy="0 0 0" xyz="0.0000 -0.39975 0.220625" />
+        <geometry>
+          <box size="0.7 0.4 0.32" />
+        </geometry>
+      </collision>
+      <collision>
+        <origin rpy="0 0 0" xyz="0.316343 0.0000 0.22" />
+        <geometry>
+          <box size="0.0673 0.4 0.32" />
+        </geometry>
+      </collision>
+      <collision>
+        <origin rpy="0 0 0" xyz="0.0000 0.0000 0.39" />
+        <geometry>
+          <box size="0.7 1.2 0.02" />
+        </geometry>
+      </collision>
+    </link>
+    <joint name="coffee_table_drawer_joint" type="prismatic">
+      <parent link="coffee_table" />
+      <child link="coffee_table_drawer" />
+      <axis xyz="-1 0 0" />
+      <limit effort="1000.0" lower="0.0" upper="0.5" velocity="0.5" />
+    </joint>
+    <link name="coffee_table_drawer">
+      <visual>
+        <geometry>
+          <mesh filename="package://iai_apartment/meshes/visual/coffee_table_drawer.dae" />
+        </geometry>
+      </visual>
+      <collision>
+        <origin rpy="0 0 0" xyz="-0.338806 0.0000 0.21892" />
+        <geometry>
+          <box size="0.0214 0.396 0.31569" />
+        </geometry>
+      </collision>
+      <collision>
+        <origin rpy="0 0 0" xyz="-0.051066 0.0000 0.107979" />
+        <geometry>
+          <box size="0.5521 0.3454 0.02144" />
+        </geometry>
+      </collision>
+      <collision>
+        <origin rpy="0 0 0" xyz="0.207974 0.0000 0.238907" />
+        <geometry>
+          <box size="0.0214 0.3486 0.2398" />
+        </geometry>
+      </collision>
+      <collision>
+        <origin rpy="0 0 0" xyz="-0.051066 -0.186206 0.226597" />
+        <geometry>
+          <box size="0.5521 0.0214 0.26425" />
+        </geometry>
+      </collision>
+      <collision>
+        <origin rpy="0 0 0" xyz="-0.051066 0.186206 0.226597" />
+        <geometry>
+          <box size="0.5521 0.0214 0.26425" />
+        </geometry>
+      </collision>
+    </link>
+    <joint name="bedside_table_joint" type="fixed">
+      <origin rpy="0 0 0" xyz="7.2110 0.1296 0.0" />
+      <parent link="middle_of_the_room" />
+      <child link="bedside_table" />
+    </joint>
+    <link name="bedside_table">
+      <visual>
+        <geometry>
+          <mesh filename="package://iai_apartment/meshes/visual/bedside_table.dae" />
+        </geometry>
+      </visual>
+      <collision>
+        <origin rpy="0 0 0" xyz="0.0000 0.0000 0.230" />
+        <geometry>
+          <box size="0.312 0.312 0.460" />
+        </geometry>
+      </collision>
+    </link>
+
+  <!-- kitchen, kitchen on the walls -->
+    <xacro:include filename="$(find iai_apartment)/urdf/kitchen.xarco" />
+    <xacro:kitchen_macro name="kitchen_base" parent="apartment_root">
+      <origin xyz="-0.05 0.75 0.005"/>
+    </xacro:kitchen_macro>
+
+  <!-- Collision okay Walls -->
```

Comment:
Please put the wall-related collision meshes next to the corresponding visual mesh, such that they are easier to find and it's clearer what happened to the wall link's collision mesh that got commented out.

Reply:
Addressed in the new commit.